addons/mail/models/discuss/discuss_channel.py

```diff
@@ -43,14 +43,15 @@ def _generate_random_token(self):
         return ''.join(choice('abcdefghijkmnopqrstuvwxyzABCDEFGHIJKLMNPQRSTUVWXYZ23456789') for _i in range(10))
 
     # description
-    name = fields.Char('Name', required=True)
+    name = fields.Char('Name', required=True, compute="_compute_name", store=True, readonly=False)
     active = fields.Boolean(default=True, help="Set active to false to hide the channel without removing it.")
     channel_type = fields.Selection([
         ('chat', 'Chat'),
         ('channel', 'Channel'),
         ('group', 'Group')],
         string='Channel Type', required=True, default='channel', readonly=True, help="Chat is private and unique between 2 persons. Group is private among invited persons. Channel can be freely joined (depending on its configuration).")
     is_editable = fields.Boolean('Is Editable', compute='_compute_is_editable')
+    auto_recompute_name = fields.Boolean(default=False, help="If True, the name of the channel will be recomputed automatically when the channel members change.")
```

Comment:
Why place the field here? I would expect it to be either close to the "name" field or alphabetically.